test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -676,10 +676,10 @@ private void BuildProjectJson(string projectDirectory)
 
         private void MigrateProject(params string[] migrateArgs)
         {
-            var result =
-                MigrateCommand.Run(migrateArgs);
+            var result = new TestCommand("dotnet")
+                    .Execute($"migrate {string.Join(" ", migrateArgs)}");
 
-            result.Should().Be(0);
+            result.Should().Pass();
```

Comment:
i think we should just chain-call Should().Pass() i don't think there is much value of having the variable